pkg/ovs/ovsconfig/ovs_client.go

```diff
@@ -106,7 +106,7 @@ func (br *OVSBridge) Create() Error {
 	} else if exists {
 		klog.Info("Bridge exists: ", br.uuid)
 		// Update OpenFlow protocol versions on existent bridge.
-		if err := br.updateProtocols(); err != nil {
+		if err := br.updateProtocolsAndDatapathType(); err != nil {
```

Comment:
I feel like this could have been 2 different methods: `updateProtocols` and `updateDatapathType`. Or a single method with a different name (`updateBridgeConfiguration`).

Reply:
I opted for later to do the update in one transaction.